cloudweatherreport/templates/base.html

```diff
@@ -1,150 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <title>{{ title }}</title>
-    <link rel="stylesheet" type="text/css" media="screen" href="static/css/ubuntu-styles.css">
-    <link rel="stylesheet" type="text/css" href="static/css/base.css">
-    <script type='text/javascript' src='static/js/jquery.min.js'></script>
-    <script type="text/javascript">
-        function hide_rows(name) {
-            name = '.' + name;
-            $(name).hide();
-            return false;
-        }
-        function toggle_rows(name) {
-            name = '.' + name;
-            $(name).toggle();
-            return false;
-        }
-    </script>
+        <meta charset="UTF-8">
+        <title>{% block title %}{% endblock %}</title>
+        <!--link rel="stylesheet" type="text/css" media="screen" href="static/css/ubuntu-styles.css" -->
+        <link rel="stylesheet" type="text/css" media="screen" href="https://assets.ubuntu.com/v1/vanilla-framework-version-0.0.65.min.css">
+        <!-- link rel="stylesheet" type="text/css" media="all"
+              href="http://assets.ubuntu.com/sites/guidelines/css/latest/ubuntu-styles.css"/ -->
+        <link rel="stylesheet" type="text/css" href="http://status.juju.solutions/static/css/base.css" >
+
+    {% block head %}
+    {% endblock %}
+
 </head>
-{% macro display_status_img(value, size='30px') -%}
-    {% if value == 'PASS' or value == 'All Passed' %}
-        <span class="test-result pass">&#x2714;</span>
-    {% elif value == 'FAIL' or value == 'All Failed' %}
-        <span class="test-result fail">&#x2718;</span>
-    {% elif value == 'Some Failed' %}
-        <span class="test-result fail">&#x2718;</span>
-    {% elif value == 'Provisioning Failed' %}
-        <span class="test-result cloud-fail">&#x25B2;</span>
-    {% else %}
-        <span class="test-result no-result">&#x26AB;</span>
+
+
+{% macro display_pagination(pagination) -%}
+    {% if pagination.has_prv %}
+        <div class="button--secondary">
+            <a href="{{ pagination.prv_page }}">&#171; Prv</a>
+        </div>
+    {% endif %}
+    {% if pagination.has_next %}
+        <div class="button--secondary">
+            <a href="{{ pagination.next_page }}">Next &#187;</a>
+        </div>
     {% endif %}
 {%- endmacro %}
-<body class="details-page">
-<div class="container">
 
-    <div class="align-center">
-        <h3>Cloud Weather Report</h3>
-    </div>
+<body class="homepage">
+
+<header class="banner global" role="banner">
+    <nav role="navigation" class="nav-primary nav-right" id="nav">
+        <div class="logo">
+            <a href="/"><span>Cloud Weather Report</span></a>
+        </div>
+        <ul>
+            <li>
+                <a href="/bundles">Bundles</a>
```

Comment:
@chuckbutler This hasn't been deployed yet, so that page is still accurate as far as that site goes.

@seman @chuckbutler Is that page useful?  I didn't actually notice that it existed until now, so I missed generating it, but if it's useful then perhaps I should fix that.